atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -25,18 +25,24 @@ class BumpReleasePlugin(PreBuildPlugin):
     # The target parameter is no longer used by this plugin. It's
     # left as an optional parameter to allow a graceful transition
     # in osbs-client.
-    def __init__(self, tasker, workflow, hub, target=None):
+    def __init__(self, tasker, workflow, hub, target=None, koji_ssl_certs_dir=None):
         """
         constructor
 
         :param tasker: DockerTasker instance
         :param workflow: DockerBuildWorkflow instance
         :param hub: string, koji hub (xmlrpc)
         :param target: unused - backwards compatibility
+        :param koji_ssl_certs_dir: str, path to "cert", "ca", and "serverca"
```

Comment:
I was thinking to add all koji params here - just in case this plugin would ever need a proper authentication